RecoPPS/Local/python/ctppsDiamondLocalReconstruction_cff.py

```diff
@@ -1,5 +1,13 @@
 import FWCore.ParameterSet.Config as cms
 
+# Temporary customization of GT - to be removed once the tag is included in the GTs
+from Configuration.StandardSequences.FrontierConditions_GlobalTag_cff import GlobalTag
+GlobalTag.toGet.append(
+  cms.PSet(record = cms.string('PPSTimingCalibrationLUTRcd'),
+           tag = cms.string('PPSDiamondTimingCalibrationLUT_test')
+  )
+)
+
```

Comment:
@cms-sw/alca-l2 excuse me. Why has this been tolerated?

Reply:
Hi @perrotta I had a private discussion with Marco and realized that this solution create issues when running without having access to CondDB, which are pretty rare cases (open-data, HPC...). Now that we have "fixed" the isssues with Geometry and EGM tags and prepared MWGR we'll start working on this.